doc/specs/stdlib_io.md

```diff
@@ -131,3 +131,130 @@ program demo_savetxt
     call savetxt('example.dat', x) 
 end program demo_savetxt
 ```
+
+## `disp` - display your data
+
+### Status
+
+Experimental
+
+### Class
+
+Impure subroutine.
+
+### Description
+
+Outputs a `logical/integer/real/complex/character/string_type` scalar or `logical/integer/real/complex` and rank-1/rank-2 array to the screen or a file `unit`.
+
+#### More details
+
+```fortran
+call disp( A(i, j, 2, :, 1:10) [, header, unit, brief] )    !! `i, j, ...` can be determined by `do` loop.
+```
+
+For `complex` type, the output format is `*(A25, 1X)`; 
+For    other types, the output format is `*(A12, 1X)`.
+
+To prevent users from accidentally passing large-length arrays to `disp`, causing unnecessary io blockage:
+1. If the `brief` argument is not specified, `disp` will print **the brief array content with a length of `10*50` by default**.
```

Comment:
I am not in favour of this behaviour. I would expect that when brief is not provided, then the defualt is false.

Reply:
This approach is reasonable and achievable. `10*50` is also sufficient for general needs. My concern mainly stems from the fact that Fortran is a compiled language and users may pass large arrays to `disp`.
Of course, this can be unnecessary, because we can also pass a large array to `print` in normal usage:
```fortran
print *, x !! x(1:100000, 1:100000)
```

Reply:
This approach is reasonable and achievable. `10*50` is also sufficient for general needs. My concern mainly stems from the fact that Fortran is a compiled language and users may pass large arrays to `disp`.
Of course, this can be unnecessary, because we can also pass a large array to `print` in normal usage:
```fortran
print *, x !! x(1:100000, 1:100000)
```

Reply:
Same as comment here, https://github.com/fortran-lang/stdlib/pull/520/#discussion_r720709803
> `brief` is `.false.` by default now.